distributed/diagnostics/task_stream.py

```diff
@@ -33,6 +33,8 @@ def transition(self, key, start, finish, *args, **kwargs):
         if start == "processing":
             if key not in self.scheduler.tasks:
                 return
+            if not kwargs.get("startstops", []):
```

Comment:
No need to create an empty list here for a boolean check

```suggestion
            if not kwargs.get("startstops"):
```

Reply:
I added it as a readability hint to make it clearer that this isn't equivalent to `if not "startstops" in kwargs`. I felt like the case of the key either being missing or being present with an empty list was a little subtle. Happy to remove though.

Reply:
No strong thoughts either way. A comment as to why we return there might be more useful than any type info here.

Reply:
Went with the comment instead